parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -844,6 +844,22 @@ ensureGPUDrivers() {
     fi
 }
 
+# TODO: this is a temporary ubuntu-only HACK until we get a driver
+ensureAMDGPUDrivers() {
+    echo "Installing AMD GPU drivers"
+
+     # delete amdgpu module from blacklist
+    sudo sed -i '/blacklist amdgpu/d' /etc/modprobe.d/blacklist-radeon-instinct.conf
+
+    pushd /var/cache/amdgpu-apt
+    ls -l
+    sudo dpkg -i *.deb
```

Comment:
why do we need `autoconf automake autotools-dev` those three packages are purely to compile the drivers from scratch. are we doing that ?  

We shouldnt need to keep those... (not even sure why we need them)

Reply:
It's dependency of amdgpu. So I'm not sure what can I do with it.

I can run another set of tests without it (the feedback loop is frutrasting).